packages/core/src/App/app.jsx

```diff
@@ -13,6 +13,7 @@ import {
     setSharedMT5Text,
 } from '@deriv/shared';
 import { initializeTranslations, getLanguage, useOnLoadTranslation } from '@deriv/translations';
+import { CashierStore } from '@deriv/cashier';
```

Comment:
we lazy load `cashier` bundle in [routes-config.js](https://github.com/binary-com/deriv-app/pull/2197/files#diff-9889cd7f12e163a1f562f93d101e8ca18a4a6ab17e61961e8deed3f0f9233a70R34) but then we import it during App init phase. Is it possible to attach cashier store lazily? 
May be we can attach cashier store during Cashier App init (`packages/cashier/src/App.jsx`)

Reply:
We don't load the `cashier` package itself. there are 2 different bundles coming out of that package. One is the "react" app, (note the air quote), and the other one is the store itself. The import is being transformed using `deriv-shared-loader` custom loader to only load the store file out of that package.